app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    def dead(self) -> None:
+        if self.health <= 0:
+            self.alive.remove(self)
```

Comment:
u should to use class name instead of self)

Reply:
ok, but could you explain why?)

Reply:
`couse alive its class atribute but not object of class)